source/compatibility.tex

```diff
@@ -2097,7 +2098,7 @@
 block.
 Allowing jump past initializers would require
 complicated runtime determination of allocation.
-Furthermore, any use of the uninitialized object could be a
+Furthermore, any use of the uninitialized object can be a
```

Comment:
This again seems to convey a risk: even taking the address of the uninitialized object can be a disaster!  Perhaps more formal language will help:

> Furthermore, many operations on such an uninitialized object have undefined behavior.

Reply:
Agreed, separate PR.

Reply:
I mourn the loss of our last instance of "disaster".

Reply:
We've had only that one instance since Git history began, but it is nice to finally be able to say that C++ isn't a disaster.

Reply:
At least we will always have major catastrophe.